Softeq.XToolkit.WhiteLabel.Droid/DroidBootstrapperBase.cs

```diff
@@ -33,16 +33,30 @@ protected override void RegisterInternalServices(IContainerBuilder builder)
             builder.PerDependency<DroidFrameNavigationService, IFrameNavigationService>(IfRegistered.Keep);
         }
 
-        protected override void ConfigureIoc(IContainerBuilder builder)
+        protected override IList<Assembly> SelectAssemblies()
         {
+            return new List<Assembly>
+            {
+                typeof(DroidBootstrapperBase).Assembly
+            };
+        }
+
+        protected override bool IsExtractToAssembliesCache(Type type)
+        {
+            return typeof(FragmentActivity).IsAssignableFrom(type)
+                || typeof(DialogFragment).IsAssignableFrom(type)
+                || typeof(Fragment).IsAssignableFrom(type);
         }
     }
 
     public class DroidViewModelFinder : ViewModelFinderBase
     {
         protected override IEnumerable<Type> SelectViewsTypes(Assembly assembly)
         {
-            return assembly.GetTypes().View(typeof(AppCompatActivity), typeof(Fragment), typeof(DialogFragment));
+            return assembly.GetTypes().View(
```

Comment:
Seems like a bit of a duplication here - ViewModelFinder search for same types that extracted to cache. Baybe, we can define the list of types for each platform and pass them to ViewModelFinder? This way we could also unify ViewModelFinder's logic in single class

Reply:
Yes, you're right, it's a good point!
I've added changes.